app/models/person.rb

```diff
@@ -52,11 +52,12 @@ class Person < ActiveRecord::Base
   delegate :update_location, :district, :state, to: :location
 
   FIELDS_ALSO_ON_NB = %w[email first_name last_name is_volunteer phone]
-  PERMITTED_PUBLIC_FIELDS = [:email, :phone, :first_name, :last_name, :address, :city, :state_abbrev, :zip, :is_volunteer, remote_fields: [:event_id, :employer, :occupation, :skills, tags: []]]
+  PERMITTED_PUBLIC_FIELDS = [:email, :phone, :first_name, :last_name, :full_name, :address, :city, :state_abbrev, :zip, :is_volunteer, remote_fields: [:event_id, :employer, :occupation, :skills, tags: []]]
```

Comment:
Hi,

Yes, it is handled on the nationbuilder side.  I was planning on using 
it for the new donation page, but might reverse this decision.

-Mario

On 2/27/2016 8:13 AM, Jacob Turino wrote:
>
> In app/models/person.rb 
> <https://github.com/MayOneUS/mayday-2.0-backend/pull/82#discussion_r54335191>:
>
> > @@ -52,11 +52,12 @@ class Person < ActiveRecord::Base
> >    delegate :update_location, :district, :state, to: :location
> >
> >    FIELDS_ALSO_ON_NB = %w[email first_name last_name is_volunteer phone]
> > -  PERMITTED_PUBLIC_FIELDS = [:email, :phone, :first_name, :last_name, :address, :city, :state_abbrev, :zip, :is_volunteer, remote_fields: [:event_id, :employer, :occupation, :skills, tags: []]]
> > +  PERMITTED_PUBLIC_FIELDS = [:email, :phone, :first_name, :last_name, :full_name, :address, :city, :state_abbrev, :zip, :is_volunteer, remote_fields: [:event_id, :employer, :occupation, :skills, tags: []]]
>
> As far as I can tell, |:full_name| can be passed in, but nothing will 
> happen with it if it does. Are we supposed to try to split it and save 
> it locally as first and last or just pass it directly to NB as is? Is 
> there a form on the frontend that passes in full_name?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MayOneUS/mayday-2.0-backend/pull/82/files#r54335191>.
>